### PR TITLE
Update Super ALT↯TAB to remove use of unregister_code16

### PR DESCRIPTION
**Pull request by jezcope**
## Description

The code as originally listed didn't work for me, but replacing `unregister_code16(LALT(KC_TAB));` with `unregister_code(KC_LALT);` fixes the problem and causes the macro to work as intended.

Thanks to folks on Discord for helping me figure this out.
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by fauxpark** [comment]
The specific issue has to do with how `(un)register_code16` works. If the given keycode is a modifier (eg. `KC_LALT`), or `KC_NO`, the modifier is added to the real mods. If not, it's added to the *weak* mods:

https://github.com/qmk/qmk_firmware/blob/db440f3e75f1d71e2c26499c7c7500cc48020047/quantum/quantum.c#L134-L141

 Because `LALT()` is not actually `KC_LALT` itself, the second code path is chosen and QMK attempts to clear the Left Alt bit from `weak_mods` - except that `register_code(KC_LALT)` will actually set it in `real_mods`:

https://github.com/qmk/qmk_firmware/blob/db440f3e75f1d71e2c26499c7c7500cc48020047/tmk_core/common/action.c#L766-L769

So the moral of the story is to use the matching functions when registering and unregistering keycodes.

File: docs/feature_macros.md [hunk 1]
```diff
@@ -230,7 +230,7 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
 void matrix_scan_user(void) {     # The very important timer. 
   if (is_alt_tab_active) {
     if (timer_elapsed(alt_tab_timer) > 1000) {
-      unregister_code16(LALT(KC_TAB));
+      unregister_code(KC_LALT);
```
Comment (drashna) [inline docs/feature_macros.md]: Don't forget to unregister TAB here, as well.

Comment (fauxpark) [reply docs/feature_macros.md]: The Tab key isn't being held, only tapped, so there's no need.

Comment (jezcope) [reply docs/feature_macros.md]: Yup, line 223 already has `unregister_code(KC_TAB)` whenever the magic key is released.

**Pull request merged**